docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -0,0 +1,244 @@
+
+.${buttonUnstyledClasses.disabled} { /* ".MuiButton-disabled" */
+  cursor: not-allowed;
+}
+```
+
+Here's a complete demo of using ButtonUnstyled and useButton with plain CSS:
+
+{{"demo": "BaseButtonPlainCss.js", "defaultCodeOpen": false}}
+
+### Styling with Tailwind CSS
+
+After installing Tailwind, simply pass Tailwind's utility classes to `className`.
+
+```tsx
+<ButtonUnstyled className="bg-green-600 rounded-md py-1 px-4...">
+  Create Repository
+</ButtonUnstyled>
+```
+
+Here's a complete demo of the same button styled with Tailwind instead:
+
+{{"demo": "BaseButtonTailwind.js", "hideToolbar": true}}
+
+### Styling with MUI System
+
+[MUI System](/system/getting-started/overview/) is a small set of CSS utilties that provide a styled-components-like API for building out designs that adhere to a theme.
+
+MUI System's core utility is a [`styled` function](/system/styled/) that is equivalent to emotion's or styled-components' styled(). Interpolations or arguments that are functions called by `styled` receive the `theme` from an upper `ThemeProvider`.
+
+```tsx
+import * as React from 'react';
+import { ThemeProvider } from '@emotion/react';
+import ButtonUnstyled from '@mui/base/ButtonUnstyled';
+
+const theme = {
+  colors: {
+    primary: 'green',
+  },
+};
+
+const GithubButton = styled(ButtonUnstyled)(
+  ({ theme }) => `
+    background-color: ${theme.colors.primary /* => 'green' */};
+  `,
+);
+
+render(
+  <ThemeProvider theme={theme}>
+    <GithubButton>Create Repository</GithubButton>
+  </ThemeProvider>,
+);
+```
+
+Most of the demos and examples in the Base UI documentation are styled with MUI system in this way. You can inspect the `theme` object used on this site in your browser console, or explore the default structure [here](/material-ui/customization/default-theme/).
+
+#### Styling ButtonUnstyled with MUI System
+
+```tsx
+import * as React from 'react';
+import ButtonUnstyled from '@mui/base/ButtonUnstyled';
+import { styled } from '@mui/system';
+
+const GithubButton = styled(ButtonUnstyled)(
+  ({ theme }) => `
+    background-color: ${theme.palette.mode === 'dark' ? '#238636' : '#1f883d'};
+    ${/* ... the rest of the styles */}
+  `,
+);
+
+export default function App() {
+  return (
+    <div>
+      <GithubButton>Create Repository</GithubButton>
+    </div>
+  );
+}
+```
+
+#### Styling useButton with MUI System
+
+```tsx
+import * as React from 'react';
+import useButton from '@mui/base/useButton';
+import { styled } from '@mui/system';
+
+const GithubButton = styled('button')(
+  ({ theme }) => `
+    background-color: ${theme.palette.mode === 'dark' ? '#238636' : '#1f883d'};
+    ${/* ... the rest of the styles */}
+  `,
+);
+
+export default function App() {
+  const { getRootProps } = useButton(/* props*/);
+
+  return (
+    <div>
+      <GithubButton type="button" {...getRootProps()}>
+        Create Repository
+      </GithubButton>
+    </div>
+  );
+}
+```
+
+#### Using the sx prop
```

Comment:
I think this is worth upgrading to an h3 so it sits at the same level as the other styling options.

```suggestion
### Styling with the sx prop
```